tests/integration/security/jwt_test.go

```diff
@@ -270,6 +270,84 @@ func TestRequestAuthentication(t *testing.T) {
 		})
 }
 
+func TestRequestAuthenticationWithInvalidJwks(t *testing.T) {
```

Comment:
Why not just extend TestRequestAuthentication? Then we don't need a new namespace, 2 new pods, spin everything up, tear down, etc. Adds  a lot of test time

Reply:
yes, as you're creating the new workload `f`, you can just put it in the existing TestRequestAuthentication.